includes/emails/class-give-email-tags.php

```diff
@@ -529,8 +539,18 @@ function give_email_tag_fullname( $tag_args ) {
 			break;
 
 		case give_check_variable( $tag_args, 'isset', 0, 'user_id' ):
-			$donor = new Give_Donor( $tag_args['user_id'], true );
-			$fullname  = trim( "{$donor->get_first_name()} {$donor->get_last_name()}" );
+			$donor    = new Give_Donor( $tag_args['user_id'], true );
+			$fullname = trim( "{$donor->get_first_name()} {$donor->get_last_name()}" );
+			break;
+
+		/**
```

Comment:
@raftaar1191 This type of comment block should be implemented for Functions, Actions and Filter and not for individual case of switch.

Reply:
Thanks @mehul0810 for point this